app/services/api/v3/ecf/partnerships_query.rb

```diff
@@ -20,6 +20,10 @@ def partnerships
           scope
         end
 
+        def partnership
+          lead_provider.partnerships.includes(:school, :cohort, :delivery_partner).find(params[:id])
```

Comment:
actually could we use `:cohort, :delivery_partner, school: :induction_coordinators` here and on [line](https://github.com/DFE-Digital/early-careers-framework/pull/3115/files#diff-7ac983a5a95593f7fca49240aef5ba88fe07d3f1c322980f69c21d20cd2c55baL15), it should do the trick, otherwise all good :)

Reply:
I'm not sure if there's any benefit to adding `includes` here as we're only retrieving one record.

Reply:
When we retrieve one record n=1, therefore 1+1.
I don't think it'll make it worse though.